### ISSUE TITLE
Support for large files

### ISSUE THREAD

**Issue by Anihigg** [RobinLinus/snapdrop#231]
Thanks for the tool, it was great for transferring transparent image assets to my ipad to edit. I also tried to transfer large transparent avi files but that didn't save out properly. Any ideas?

Donated 10USD for support ty so much!

**Comment by RobinLinus** [comment]
Thanks a lot! Two questions: 
1. You transferred from a PC to an iPad? 
2. What do you mean by "large files"? Around 1GB?

**Comment by RobinLinus** [comment]
Just tested Chrome -> Chrome and Safari -> Chrome on a MacBook. It worked for a ~800MB file. However, it failed for a ~2GB file. Looks like the file was fully transferred, still I could not download it. The downloads bar showed an error saying "System busy".

**Comment by Anihigg** [comment]
Yes the transfer was PC to iPad. 2.31GB was what I tried. I had similar experience, the upload was ok but when I downloaded on the ipad it took me to an unknown page and didn't download.

**Comment by dougbtv** [comment]
Also reporting the same between Chrome -> Android.

Can transfer files around 600-800mb but anything I've tried in the 1.2 gig range winds up failing for me.

**Comment by TrevCan** [comment]
Also have an issue with not-so-large files. I'm trying to transfer a `zip` 350 MB file from an iOS Device (iPhone 6s) to a Linux laptop (running ungoogled-chromium). Will inform if this persists in other web browsers.

**Comment by sivaraam** [comment]
Hi! Just wanted to note here that I was trying to transfer a file ~2.4 GB in size from an Android (Firefox) to iOS device (Chrome) but the transfer kept failing. Once it failed after ~ 10% of transfer. Next it failed after ~ 60% of the transfer. The transfer never completed.

**Comment by marufmax** [comment]
+1 for this
